### PR TITLE
Fixed description for colorizeLogEntries hook

### PR DESCRIPTION
**Pull request by Tastaturberuf**
Kann nicht gut in Englisch formulieren.

### CODE REVIEW

File: docs/dev/reference/hooks/colorizeLogEntries.md [hunk 1]
```diff
@@ -8,12 +8,9 @@ aliases:
 ---
 
 
-The `colorizeLogEntries` hook is triggered when a user closes his account. It passes 
-the user ID, the operation mode and the module as arguments and does not expect 
-a return value.
-
-The operation mode will either be `close_deactivate` or `close_delete`.
-
+The `colorizeLogEntries` hook is triggered when a log entry is saved. It passes
```
Comment (fritzmg) [inline docs/dev/reference/hooks/colorizeLogEntries.md]:
```suggestion
The `colorizeLogEntries` hook is triggered when a log entry is displayed. It passes 
```

File: docs/dev/reference/hooks/colorizeLogEntries.md [hunk 2]
```diff
@@ -8,12 +8,9 @@ aliases:
...
 
-The `colorizeLogEntries` hook is triggered when a user closes his account. It passes 
-the user ID, the operation mode and the module as arguments and does not expect 
-a return value.
-
-The operation mode will either be `close_deactivate` or `close_delete`.
-
+The `colorizeLogEntries` hook is triggered when a log entry is saved. It passes 
+the data of the log entry and the label created by Contao and does expect 
+a string for the label as return.
```
Comment (fritzmg) [inline docs/dev/reference/hooks/colorizeLogEntries.md]:
```suggestion
a string for the label as return value.
```

**Review by fritzmg — CHANGES_REQUESTED** [review]

**Review by fritzmg — APPROVED** [review]

**Comment by Tastaturberuf** [comment]
sorry has mistaken the changes und resolved without apply the commit

**Comment by fritzmg** [comment]
Hm? What do you mean? Everything looks fine.

**Comment by Tastaturberuf** [comment]
yep now. but yesterday i have solve (it close and ignore) your changes. Instead of apply them.

**Pull request merged**